src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/AssessmentService.kt

```diff
@@ -216,6 +217,8 @@ class AssessmentService(
       dueAt = null,
     )
 
+    assessment.dueAt = taskDeadlineService.getDeadline(assessment)
```

Comment:
we're also going to need to reset the countdown when the task (assessment, placement request, match request) is **reallocated** to a new person.  See https://dsdmoj.atlassian.net/browse/APS-168?focusedCommentId=427694

Reply:
Ah nice, that's one less ticket to worry about!